### PR TITLE
Introducing basic system test for Bigtable.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: system_tests/bigtable.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    def setUp(self):
+        self.clusters_to_delete = []
+
+    def tearDown(self):
+        for cluster in self.clusters_to_delete:
+            cluster.delete()
+
+    def test_list_zones(self):
+        zones = CLIENT.list_zones()
+        self.assertEqual(sorted(zones), list(EXPECTED_ZONES))
```
Comment (tseaver) [inline system_tests/bigtable.py]: Should probably use `sorted` for the RHS, too.

**Comment by tseaver** [comment]
Hmm, I don't get the Travis failure:  we should be skipping system-tests for PRs, shouldn't we?

**Comment by dhermes** [comment]
The Travis failure is my fault. I had to re-run the job before I remembered how things work. `system_tests/bigtable.py` will get imported by `attempt_system_tests` so the client set-up should happen at run-time not at import-time. D'oh!

**Comment by dhermes** [comment]
@tseaver PTAL

**Comment by jonparrott** [comment]
LGTM.

**Comment by dhermes** [comment]
Arrrg! I committed a `Review changes (will be squashed).` commit message by mistake.

**Comment by jonparrott** [comment]
Nobody is perfect.

**Comment by dhermes** [comment]
Lulz

**Pull request merged**